docs/SETUP.md

```diff
@@ -1,75 +1,106 @@
+
+### Standalone deployment
+
+Start a single containerized instance of Splunk Enterprise with the command below, replacing `<password>` with a password string that conforms to the [Splunk Enterprise password requirements](https://docs.splunk.com/Documentation/Splunk/latest/Security/Configurepasswordsinspecfile).
+
+```
+$ docker run -it -p 8000:8000 -e "SPLUNK_PASSWORD=<password>" -e "SPLUNK_START_ARGS=--accept-license" splunk/splunk:latest
+```
+
+**You successfully created a standalone deployment with `docker-splunk`!**
+
+After the container starts up, you can access Splunk Web at <http://localhost:8000> with `admin:<password>`.
+
+### Distributed deployment
+
+Start a Splunk Universal Forwarder running in a container to stream logs to a Splunk Enterprise standalone instance, also running in a container.
+
+First, create a [network](https://docs.docker.com/engine/reference/commandline/network_create/) to enable communication between each of the services.
+
 ```
 $ docker network create --driver bridge --attachable skynet
 ```
 
-##### Splunk Enterprise
-Use the following command to start a single standalone instance of Splunk Enterprise:
+#### Splunk Enterprise
+Start a single, standalone instance of Splunk Enterprise in the network created above, replacing `<password>` with a password string that conforms to the [Splunk Enterprise password requirements](https://docs.splunk.com/Documentation/Splunk/latest/Security/Configurepasswordsinspecfile).
 ```
-$ docker run --network skynet --name so1 --hostname so1 -p 8000:8000 -e "SPLUNK_PASSWORD=<password>" -e "SPLUNK_START_ARGS=--accept-license" -it splunk/splunk:latest
+$ docker run -it --network skynet --name so1 --hostname so1 -p 8000:8000 -e "SPLUNK_PASSWORD=<password>" -e "SPLUNK_START_ARGS=--accept-license" splunk/splunk:latest
 ```
 
-Let's break down what this command does:
-1. Start a Docker container using the `splunk/splunk:latest` image
-2. Launch the container in the formerly-created bridge network `skynet`
-3. Name the container + hostname as `so1`
-4. Expose a port mapping from the host's `8000` to the container's `8000`
-5. Specify a custom `SPLUNK_PASSWORD` - be sure to replace `<password>` with any string that conforms to the [Splunk Enterprise password requirements](https://docs.splunk.com/Documentation/Splunk/latest/Security/Configurepasswordsinspecfile)
-6. Accept the license agreement with `SPLUNK_START_ARGS=--accept-license` - this must be explicitly accepted on every container, otherwise Splunk will not start
+This command does the following:
+1. Starts a Docker container using the `splunk/splunk:latest` image.
+1. Launches the container in the formerly-created bridge network `skynet`.
+1. Names the container and the host as `so1`.
+1. Exposes a port mapping from the host's `8000` port to the container's `8000` port
+1. Specifies a custom `SPLUNK_PASSWORD`.
```

Comment:
Does this generate a numbered list? I'm seeing all `1.` in the source but the rich diff is showing me otherwise and I don't know what's going on :P

Reply:
It'll automatically render the numbering. Using `1.` for all of them makes it easier to edit lists later without fiddling with the specific numbers.